sensor/models.py

```diff
@@ -189,15 +189,14 @@ def get_rawdata(self):
 
 
 class RawData(Model):
-    sensor_id = CharField(max_length=128)
-    s_id = ForeignKey(Sensor)
+    sensor = ForeignKey(Sensor)
     timestamp_recieved = DateTimeField()
     timestamp_data = DateTimeField()
     value = FloatField(default=-1)
     processed = BooleanField(default=False)
 
     def __unicode__(self):
-        return "Sensor:%s: Value:%s" % (self.sensor_id, self.value)
+        return "Sensor:%s: Value:%s" % (self.sensor_string_id, self.value)
```

Comment:
Who has a property named `sensor__string_id`?  I thought you deleted it in Migration 60?